### PR TITLE
docs update & folder structure rationalization

### PR DESCRIPTION
**Pull request by romsok24**
The changes in this PR are related to updating the documentation part of CBS 
and to simplifying the project folder structure.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,47 +2,47 @@
 
 ## Design
 
-We decided to go with separate tool for building (Continuous Integration) and deploying (Continuous Deployment).
+Central Build System (CBS) is made up of two modern software components: one for building (*Continuous Integration*) and one for deploying (*Continuous Deployment*).
 
-For building we choose [tekton](tekton.dev), for deploying [ArgoCD](https://argoproj.github.io/argo-cd/).
-You can find more information in [Design Doc](https://github.com/epiphany-platform/epiphany/pull/1243).
+For building we chose [tekton](tekton.dev) and for deploying - [ArgoCD](https://argoproj.github.io/argo-cd/).
+You can find more information in [Comparision of CI/CD tools](https://github.com/pprach/epiphany/blob/440c276cf22d873cc91478af4777ef714c8c1642/docs/design-docs/cicd-server/comparision_cicd.md) doc.
```
Comment (rafzei) [inline README.md:8]: I think we shouldn't point to any fork in our documentation. Can we move this comparison to our repo?

File: docs/CBS_quickstart.md [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+    - [method b) Connect from localhost through VPN](#method-b-connect-from-localhost-through-vpn)
+  - [Create Azure credentials](#create-azure-credentials)
+  - [Install CBS itself](#install-cbs-itself)
+    - [Terraform state](#terraform-state)
+    - [Terraform vars](#terraform-vars)
+    - [Azure credentials](#azure-credentials)
+    - [Running terraform code](#running-terraform-code)
+  - [DNS names](#dns-names)
+
+<br>
```
Comment (mkyc) [inline docs/CBS_quickstart.md:15]: why putting HTML tags in Markdown?

**Review by mkyc — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (romsok24) [reply README.md:8]: Ok. Will take it into discussion. Referenced in [#27](https://github.com/epiphany-platform/central-build-system/issues/27)

Comment (romsok24) [reply README.md:8]: Ok. Will take it into discussion. Referenced in [#27](https://github.com/epiphany-platform/central-build-system/issues/27)

File: docs/CBS_quickstart.md (hunk 2 above)
Comment (romsok24) [reply docs/CBS_quickstart.md:15]: It's well explained [here](https://www.markdownguide.org/basic-syntax/#line-break-best-practices).

**Pull request merged**